server/build-plugins.js

```diff
@@ -13,6 +13,15 @@ const logger = require('./logger');
 const config = new Config('bpanel');
 config.load({ env: true, argv: true, arg: true });
 const pluginsConfig = resolve(config.prefix, 'config.js');
+const modulesDirectory = resolve(__dirname, '../node_modules');
+
+// location of app configs and local plugins
+// defaults to ~/.bpanel/
+// this is set using bcfg at runtime with server
+// and passed through via webpack to this script
+const homePrefix = process.env.BPANEL_PREFIX
+  ? process.env.BPANEL_PREFIX
+  : resolve(os.homedir(), '.bpanel');
```

Comment:
Could simplify this as `process.env.BPANEL_PREFIX || resolve(os.homedir(), '.bpanel');`